HomeWork1PolishchukDariia.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cmath>
+#include <iostream>
+using namespace std;
+
+int main() {
+  setlocale(LC_ALL, "Russian");
+
+  double a{};
+  double b{};
+  double c{};
+  const int min{0};
+  const int max{1000};
+
+  while (true) {
+    cout << "Введите значение а от 1 до 1000:";
+    cin >> a;
+    if (a > min && a < max) {
+      break;
+    } else {
+      cout << "Неправильный диапазон! Введите число от 1 до 1000\n";
```

Comment:
```suggestion
    if (a > min && a < max) {
      break;
    } 
     cout << "Неправильный диапазон! Введите число от 1 до 1000\n";
```
`else` тут не нужен так как после оператора `break` не будет ничего выполнятся, т.к. он прерывает выполнение тела цикла это как `return` в функциях